api/Controllers/AuthController.cs

```diff
@@ -52,10 +52,10 @@ public async Task<IActionResult> Logout()
             var forwardedPort = HttpContext.Request.Headers["X-Forwarded-Port"];
 
             //We are always sending X-Forwarded-Port, only time we aren't is when we are hitting the API directly. 
-            var baseUri = HttpContext.Request.Headers.ContainsKey("X-Forwarded-Host") ? $"{HttpContext.Request.Headers["X-Base-Href"]}logout" : "/api";
```

Comment:
This appears to have been copy/pasted from sheriff-scheduling, but doesn't work for us at the moment since we don't have a /logout page on the frontend.